galite-core/src/main/kotlin/org/kopi/galite/form/dsl/FormBlock.kt

```diff
@@ -425,11 +436,14 @@ open class FormBlock(var buffer: Int, var visible: Int, ident: String, val title
         //TODO ------------end-----------
 
         super.source = source ?: sourceFile
+        super.name = ident
         super.bufferSize = buffer
         super.pageNumber = this@FormBlock.pageNumber
         super.maxRowPos = this@FormBlock.maxRowPos
         super.maxColumnPos = this@FormBlock.maxColumnPos
-        super.name = ident
+        this@FormBlock.dropListMap.forEach{
+                  super.dropListMap.put(it.key,it.value)
+        }
```

Comment:
We probably don't need this , we will recheck after implementing the other corrections